packages/mgt-components/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -693,6 +695,7 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
 
     return html`
        <div class="${classMap(inputClasses)}">
+         <span class="search-icon">${getSvg(SvgIcon.Search)}</span>
```

Comment:
Adding this comment as close to the place as I can.
When I use my keyboard to give focus to the input here I get an incomplete state for the inital render:
![image](https://user-images.githubusercontent.com/7122716/186265866-4abdb356-8c65-4e82-840f-44c00c33dfc6.png)

I feel that this experience should be the same for a keyboard user tabbing to the input as it is for a mouse user clicking in the input, likely this is an existing behavior, but we should make it better while we have the opportunity

Reply:
I made this show the dropdown when you focus in with tab key.